src/app/feed/feed-not-found/feed-not-found.component.html

```diff
@@ -11,5 +11,7 @@
 			<li>Try different keywords, accounts or hashtags.</li>
 			<li>Try more general keywords, accounts or hashtags.</li>
 		</ul>
+		<p>If you encountered an error of the system, please submit a bug report in the
+			<a href="https://github.com/fossasia/loklak_search/issues/">loklak issue tracker here</a>.</p>
```

Comment:
Make separate PR's Don't make changes related to multiple issues at once, Please make a separate PR for this.